runtime/common/src/apis/loans.rs

```diff
@@ -32,3 +34,17 @@ decl_runtime_apis! {
 		fn portfolio_valuation(pool_id: PoolId, input_prices: PriceCollectionInput) -> Result<Balance, DispatchError>;
 	}
 }
+
+pub fn update_nav_api_call<T: pallet_loans::Config>(
```

Comment:
NIT. We're using this for the API, but it seems like a common method that can be used for other purposes. I would move to `runtime/common/src/lib.rs` instead.

Reply:
Same name?

Reply:
No strong opinion. Maybe `infallible_update_nav()` or similar?